SConscript

```diff
@@ -1,6 +1,16 @@
+import platform
+
 # panda fw
 SConscript('board/SConscript')
 
 # test files
 if GetOption('test'):
+  CC = 'gcc'
+  platform = platform.system()
+  if platform == 'Darwin':
+    # gcc installed by homebrew has version suffix (e.g. gcc-12) in order to be  
+    # distinguishable from system one - which acts as a symlink to clang
+    CC += '-12'
```

Comment:
is there a more robust way to specify the brew version of gcc?

Reply:
it does not do that (in [formula](https://github.com/Homebrew/homebrew-core/blob/78c6f5303d1b91f0a09d0882835ef62791e661f0/Formula/gcc.rb), every installed gcc binary has a version suffix), so it could be distinguished from gcc/clang supplied by Xcode.

Reply:
I will have to take a look one more time to confirm, but I believe the suffixes are also there in the formula install directory at `brew --prefix gcc`